### PR TITLE
Fix Collection namespaces

### PR DESCRIPTION
**Pull request by willsb**
They kinda messy

I also reduced the number of collections so that we’re only using the absolute must. This will be needed for a future PR

Squash whenever

### CODE REVIEW

File: Toggl.Daneel/ViewSources/StartTimeEntryTableViewSource.cs [hunk 1]
```diff
@@ -28,6 +31,13 @@ public StartTimeEntryTableViewSource(UITableView tableView)
             tableView.RegisterNibForCellReuse(StartTimeEntryEmptyViewCell.Nib, emptySuggestionIdentifier);
         }
 
+        public override UIView GetViewForHeader(UITableView tableView, nint section)
+        {
+            if (!UseGrouping) return null;
+
+            return null;
```
Comment (arturs105) [inline Toggl.Daneel/ViewSources/StartTimeEntryTableViewSource.cs]: wat? I assume this for the grouping in the future. I think this method should be added, when the grouping will be introduced.

File: Toggl.Foundation.MvvmCross/Collections/WorkspaceGroupedCollection.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using MvvmCross.Core.ViewModels;
+using Toggl.Multivac;
+
+namespace Toggl.Foundation.MvvmCross.Collections
+{
+    public class WorkspaceGroupedCollection<T> : MvxObservableCollection<T>
```
Comment (arturs105) [inline Toggl.Foundation.MvvmCross/Collections/WorkspaceGroupedCollection.cs]: Shouldn't this be sealed?

File: Toggl.Daneel/ViewSources/StartTimeEntryTableViewSource.cs (hunk 1 above)
Comment (willsb) [reply Toggl.Daneel/ViewSources/StartTimeEntryTableViewSource.cs]: Agreed

File: Toggl.Foundation.MvvmCross/Collections/WorkspaceGroupedCollection.cs (hunk 2 above)
Comment (willsb) [reply Toggl.Foundation.MvvmCross/Collections/WorkspaceGroupedCollection.cs]: The idea is that on the next PR we'll inherit from it

**Pull request merged**